lib/tasks/mastodon.rake

```diff
@@ -356,6 +356,16 @@ namespace :mastodon do
 
       prompt.say "\n"
 
+      ip_retention_period_days = prompt.ask('How long should we store IP addresses in the database, for moderation purposes? (in days)') do |q|
+        q.required true
+        q.default 365
+        q.convert :int
+      end
+
+      env['IP_RETENTION_PERIOD'] = (ip_retention_period_days * 86_400)
```

Comment:
The linter wants `86_400` format here.

Separately - you could probably do `ip_retention_period_days.days.to_i`

Reply:
I like `ip_retention_period_days.days.to_i` a lot more, let's do that. Tested and it works as expected